### PR TITLE
Update docs maintainers

### PR DESCRIPTION
**Pull request by thaJeztah**
ping @SvenDowideit @sfsmithcha @vieux

### CODE REVIEW

**Comment by dongluochen** [comment]
LGTM

File: MAINTAINERS [hunk 1]
```diff
@@ -23,11 +23,10 @@
 		]
 
 	[Org."Docs maintainers"]
-
-	# TODO Describe the docs maintainers role.
-
 		people = [
-			"moxiegirl"
+			"sfsmithcha",
+			"sven",
```
Comment (allencloud) [inline MAINTAINERS]: Should this place be GitHub ID, then `SvenDowideit `?

Comment (thaJeztah) [inline MAINTAINERS]: This is the "handle" we use for @SvenDowideit in other repositories, so I sticked with that.

w.r.t. the GitHub ID; his GitHub handle is with uppercase `S` and `D`, so we use that as well (although on GitHub, it is case-insensitive - not sure if the GitHub API is case-insensitive as well)

**Comment by abronan** [comment]
LGTM

**Pull request merged**